source/viz.md

```diff
@@ -1141,9 +1233,9 @@ question we asked was only about the largest landmasses; let's make the plot a
 little bit clearer by keeping only the largest 12 landmasses. We do this using
 the `nlargest` function; the first argument is the number of rows we want and
 the second is the name of the column we want to use for comparing who is
-largest.  Then to help us make sure the labels have enough
-space, we'll use horizontal bars instead of vertical ones. We do this by
-swapping the `x` and `y` variables.
+largest. Then to help make the landmass labels easier to read
+we'll swap the `x` and `y` variables,
+so that the landmass labels are on the y-axis and we don't have to tilt our head to read them.
```

Comment:
"landmass" isn't a word, it's a variable name -- I would just say "labels" rather than "landmass labels" anyway

Reply:
apparently it is a "newer spelling" and "land mass" is the more traditional. i guess it doesn't matter much, maybe we can just keep landmass